lambdas/db-migration/src/migrations/20200813174231_create_collections_table.ts

```diff
@@ -3,19 +3,27 @@ import * as Knex from 'knex';
 export const up = async (knex: Knex): Promise<void> =>
   knex.schema.createTable('collections', (table) => {
     table.increments('cumulusId').primary();
-    table.string('name').notNullable();
-    table.string('version').notNullable();
-    table.string('sampleFileName').notNullable();
-    table.text('granuleIdValidationRegex').notNullable();
-    table.text('granuleIdExtractionRegex').notNullable();
-    table.jsonb('files').notNullable();
-    table.string('process');
-    table.string('url_path');
-    table.enum('duplicateHandling', ['error', 'replace', 'skip', 'version']);
-    table.boolean('reportToEms');
-    table.boolean('ignoreFilesConfigForDiscovery');
-    table.jsonb('meta');
-    table.jsonb('tags');
+    table.text('name').comment('Collection short_name registered with the CMR').notNullable();
+    table.text('version').comment('The version registered with the CMR').notNullable();
+    table.text('sampleFileName').comment('Example filename for this collection').notNullable();
+    table.text('granuleIdValidationRegex').notNullable().comment(
+      'The regular expression used to validate the granule ID extracted from filenames '
```

Comment:
I think this would be more readable if each `.xxx()` were on a separate line and the comment was one long line, rather than being split by `+`.

Reply:
8550c165b